### PR TITLE
chore: add model monitoring SDK code samples to model-builder

### PR DESCRIPTION
**Pull request by rosiezou**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-aiplatform/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

**Review by yil532 — APPROVED** [review]
LGTM

**Review by yil532 — APPROVED** [review]
LGTM

File: samples/model-builder/create_model_monitoring_job_example.py [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from google.cloud import aiplatform
+from google.cloud.aiplatform import model_monitoring
+from typing import Optional, Dict, Union, List
+
+
+#  [START aiplatform_sdk_create_model_monitoring_job_example]
+def create_model_monitoring_job_example(
```
Comment (dizcology) [inline samples/model-builder/create_model_monitoring_job_example.py:21]: Why are we exposing all these arguments in the sample function?  The sample function’s body should illustrate a use case of a library method, and the sample function and its arguments are there for sample testing purposes, and we are not expecting the user to run the sample function.

We expect the illustrated library method to have sensible defaults for most of its arguments.  As such the sample function should expose only what is needed at sample testing time. As is right now the sample function appears to be doing nothing but flattening some of the parameters.

**Review by dizcology — CHANGES_REQUESTED** [review]

File: samples/model-builder/create_model_monitoring_job_example.py (hunk 1 above)
Comment (dizcology) [reply samples/model-builder/create_model_monitoring_job_example.py:21]: Here is an example where the sample function exposes only what’s needed for sample testing and the sample function body illustrates a use case: https://github.com/googleapis/python-aiplatform/blob/main/samples/snippets/job_service/create_custom_job_sample.py

**Comment by rosiezou** [comment]
closing this for now. will revisit at a later time.

**Pull request closed without merging**